modules/uid2IdSystem.md

```diff
@@ -1,264 +1,84 @@
-| params.phoneHash | When you have user's hashed, normalized phone number  | Yes                                  | Yes                            |
-
-
-*Note that setting params.email will normalize email addresses, but params.phone requires phone numbers to be normalized.*
-
-Refer to [Normalization and Encoding](#normalization-and-encoding) for details on email address normalization, SHA-256 hashing and Base64 encoding.
-
-### CSTG example
-
-Configuration:
-```
-pbjs.setConfig({
-    userSync: {
-        userIds: [{
-            name: 'uid2',
-            params: {
-               serverPublicKey: '...server public key...',
-               subscriptionId: '...subcription id...',
-               email: 'user@email.com',
-               //phone: '+0000000',
-               //emailHash: '...email hash...',
-               //phoneHash: '...phone hash ...'
-            }
-        }]
-    }
-});
-```
-
-## Client Refresh mode
-
-This is the recommended mode for most scenarios. In this mode, the full response body from the UID2 Token Generate or Token Refresh endpoint must be provided to the module. As long as the refresh token remains valid, the module will refresh the advertising token as needed.
-
-To configure the module to use this mode, you must **either**:
-1. Set `params.uid2Cookie` to the name of the cookie which contains the response body as a JSON string, **or**
-2. Set `params.uid2Token` to the response body as a JavaScript object.
-
-The `uid2Cookie` param was originally `uid2ServerCookie`. The old name can still be used, however the inclusion of the word 'server' was causing some confusion. If both values are provided, `uid2ServerCookie` will be ignored.
-
-### Client refresh cookie example
-
-In this example, the cookie is called `uid2_pub_cookie`.
-
-Cookie:
-```
-uid2_pub_cookie={"advertising_token":"...advertising token...","refresh_token":"...refresh token...","identity_expires":1684741472161,"refresh_from":1684741425653,"refresh_expires":1684784643668,"refresh_response_key":"...response key..."}
-```
-
-Configuration:
-```
-pbjs.setConfig({
-    userSync: {
-        userIds: [{
-            name: 'uid2',
-            params: {
-                uid2Cookie: 'uid2_pub_cookie'
-            }
-        }]
-    }
-});
-```
-
-### Client refresh uid2Token example
-
-Configuration:
-```
-pbjs.setConfig({
-    userSync: {
-        userIds: [{
-            name: 'uid2',
-            params: {
-                uid2Token: {
-                    'advertising_token': '...advertising token...',
-                    'refresh_token': '...refresh token...',
-                    // etc. - see the Sample Token below for contents of this object
-                }
-            }
-        }]
-    }
-});
-```
+Depending on access to DII, there are two methods to generate UID2 tokens for use with Prebid.js, as shown in the following table.
```

Comment:
FOr this generic page, people probably dunno what DII is, maybe state the full name Directly Identifiable Information with the acronym and add a reference to DII defined 
https://unifiedid.com/docs/ref-info/glossary-uid
?